packages/studio-ui/src/web/components/Layout/LangServerHealthWarning.tsx

```diff
@@ -6,7 +6,7 @@ import React, { useEffect } from 'react'
 export default () => {
   useEffect(() => {
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
-    axios.get(`${window.BOT_API_PATH}/mod/nlu/health`).then(({ data }) => {
+    axios.get(`${window.STUDIO_API_PATH}/nlu/health`).then(({ data }) => {
```

Comment:
Shouldn't the health be on the runtime? The studio only trains a model. It shouldn't care about its health

Reply:
I didnt think about that 🤔

This could work better, I have to think about it